tests/backend/test_rpc_handshake.py

```diff
@@ -227,10 +231,17 @@ async def _test_authenticated_handshake_user_revoked(
     assert rep.headers["Api-Version"] == str(API_VERSION)  # This header must always be present !
 
 
+async def _test_invited_handshake_invitation_token_not_found(client: InvitedRpcApiClient):
+    rep = await client.send_ping(check_rep=False, extra_headers={"Invitation-Token": None})
+    assert rep.status_code == 415
```

Comment:
Should we defined a constant for `415` and the others one ?

Reply:
Can you open an issue for this ?

Reply:
Done

Reply:
https://github.com/Scille/parsec-cloud/issues/4121